src/test/javascript/spec/component/shared/sidebar/sidebar-accordion.component.spec.ts

```
@@ -72,11 +68,19 @@ describe('SidebarAccordionComponent', () => {
 
     it('should toggle collapse state when group header is clicked', () => {
-        const groupHeader = debugElement.query(By.css('#test-accordion-item-header-current'));
-        groupHeader.triggerEventHandler('click', null);
+        const groupKey = 'current';
+        const initialCollapseState = component.collapseState[groupKey];
+
+        component.searchValue = '';
         fixture.detectChanges();
 
-        expect(component.collapseState['current']).toBeTrue();
+        const headerElement: DebugElement = fixture.debugElement.query(By.css('#test-accordion-item-header-' + groupKey));
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)